### PR TITLE
Update windows-11-plan.md

### PR DESCRIPTION
**Pull request by RAJU2529**
1. added Windows 11 update history page link  **https://support.microsoft.com/topic/windows-11-version-21h2-update-history-a19cd327-b57f-44b9-84e0-26ced7109ba9**

2. made few words to bold
2. Corrected grammatical errors.

article link
**https://learn.microsoft.com/en-us/windows/whats-new/windows-11-plan?toc=%2Fwindows%2Fdeployment%2Ftoc.json&bc=%2Fwindows%2Fdeployment%2Fbreadcrumb%2Ftoc.json**

### CODE REVIEW

File: windows/whats-new/windows-11-plan.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ Along with user experience and security improvements, Windows 11 introduces enha
...
 - Enterprise and Education editions of Windows 11 will be supported for 36 months from the general availability date.
 
-When Windows 11 reaches general availability, a consolidated Windows 11 update history will be available on support.microsoft.com, similar to what is [available today for Windows 10](https://support.microsoft.com/topic/windows-10-update-history-1b6aac92-bf01-42b5-b158-f80c6d93eb11). Similarly, the [Windows release health](/windows/release-health/) hub will offer quick access to Windows 11 servicing announcements, known issues, and safeguard holds. 
+When Windows 11 reaches general availability, a consolidated [Windows 11 update history](https://support.microsoft.com/en-us/topic/windows-11-version-21h2-update-history-a19cd327-b57f-44b9-84e0-26ced7109ba9) is available, similar to what is [available today for Windows 10](https://support.microsoft.com/topic/windows-10-update-history-1b6aac92-bf01-42b5-b158-f80c6d93eb11). Similarly, the [Windows release health](/windows/release-health/) hub will offer quick access to Windows 11 servicing announcements, known issues, and safeguard holds.
```
Comment (JohanFreelancer9) [inline windows/whats-new/windows-11-plan.md:94]:
```suggestion
When Windows 11 reaches general availability, a consolidated [Windows 11 update history](https://support.microsoft.com/topic/windows-11-version-21h2-update-history-a19cd327-b57f-44b9-84e0-26ced7109ba9) is available, similar to what is [available today for Windows 10](https://support.microsoft.com/topic/windows-10-update-history-1b6aac92-bf01-42b5-b158-f80c6d93eb11). Similarly, the [Windows release health](/windows/release-health/) hub will offer quick access to Windows 11 servicing announcements, known issues, and safeguard holds. 
```

File: windows/whats-new/windows-11-plan.md [hunk 2]
```diff
@@ -105,7 +105,7 @@ If you run into compatibility issues or want to ensure that your organization's
...
 
-- **Test Base for Microsoft 365**: [Test Base](https://portal.azure.com/?feature.Win11=true%2F#view/Microsoft_Azure_Marketplace/GalleryItemDetailsBladeNopdl/id/Microsoft.TestBase) is a cloud testing platform that can help you evaluate applications readiness for Windows 11 upgrade or migration. IT administrators, software publishers, and system integrators can find desired test cases throughout the entire Windows validation lifecycle. Tests include, but aren't limited to, in-place upgrade validation, monthly quality updates, and tests against Windows preview features.
+- **Test Base for Microsoft 365**: [Test Base](https://portal.azure.com/?feature.Win11=true%2F#view/Microsoft_Azure_Marketplace/GalleryItemDetailsBladeNopdl/id/Microsoft.TestBase) is a cloud testing platform that can help you evaluate applications readiness for Windows 11 upgrade or migration. IT administrators, software publishers, and system integrators can find desired test cases throughout the entire Windows validation lifecycle. Tests include but aren't limited to, in-place upgrade validation, monthly quality updates, and tests against Windows preview features.
```
Comment (JohanFreelancer9) [inline windows/whats-new/windows-11-plan.md:108]:
```suggestion
- **Test Base for Microsoft 365**: [Test Base](https://portal.azure.com/?feature.Win11=true%2F#view/Microsoft_Azure_Marketplace/GalleryItemDetailsBladeNopdl/id/Microsoft.TestBase) is a cloud testing platform that can help you evaluate applications readiness for Windows 11 upgrade or migration. IT administrators, software publishers, and system integrators can find desired test cases throughout the entire Windows validation lifecycle. Tests include, but aren't limited to, in-place upgrade validation, monthly quality updates, and tests against Windows preview features.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

File: windows/whats-new/windows-11-plan.md [hunk 3]
```diff
@@ -91,7 +91,7 @@ Along with user experience and security improvements, Windows 11 introduces enha
...
 - Enterprise and Education editions of Windows 11 will be supported for 36 months from the general availability date.
 
-When Windows 11 reaches general availability, a consolidated Windows 11 update history will be available on support.microsoft.com, similar to what is [available today for Windows 10](https://support.microsoft.com/topic/windows-10-update-history-1b6aac92-bf01-42b5-b158-f80c6d93eb11). Similarly, the [Windows release health](/windows/release-health/) hub will offer quick access to Windows 11 servicing announcements, known issues, and safeguard holds. 
+When Windows 11 reaches general availability, a consolidated [Windows 11 update history](https://support.microsoft.com/topic/windows-11-version-21h2-update-history-a19cd327-b57f-44b9-84e0-26ced7109ba9) is available, similar to what is [available today for Windows 10](https://support.microsoft.com/topic/windows-10-update-history-1b6aac92-bf01-42b5-b158-f80c6d93eb11). Similarly, the [Windows release health](/windows/release-health/) hub will offer quick access to Windows 11 servicing announcements, known issues, and safeguard holds.
```
Comment (mestew) [inline windows/whats-new/windows-11-plan.md:94]:
```suggestion
When Windows 11 reaches general availability, a consolidated [Windows 11 update history](https://support.microsoft.com/topic/59875222-b990-4bd9-932f-91a5954de434) is available, similar to what is [available today for Windows 10](https://support.microsoft.com/topic/windows-10-update-history-1b6aac92-bf01-42b5-b158-f80c6d93eb11). Similarly, the [Windows release health](/windows/release-health/) hub will offer quick access to Windows 11 servicing announcements, known issues, and safeguard holds. 
```

File: windows/whats-new/windows-11-plan.md [hunk 4]
```diff
@@ -36,17 +36,17 @@ If you're looking for ways to optimize your approach to deploying Windows 11, or
...
 
-Microsoft is currently developing analysis tools to help you evaluate your devices against the Windows 11 hardware requirements. When Windows 11 reaches general availability, users running Windows 10 Home, Pro, and Pro for Workstations will be able to use the [PC Health Check](https://www.microsoft.com/windows/windows-11#pchealthcheck) app to determine their eligibility for Windows 11. Users running Windows 10 Enterprise and Education editions should rely on their IT administrators to let them know when they're eligible for the upgrade.  
+Microsoft is currently developing analysis tools to help you evaluate your devices against the Windows 11 hardware requirements. When Windows 11 reaches general availability, users running **Windows 10 Home, Pro,** and **Pro for Workstations** will be able to use the [PC Health Check](https://www.microsoft.com/windows/windows-11#pchealthcheck) app to determine their eligibility for Windows 11. Users running Windows 10 Enterprise and Education editions should rely on their IT administrators to let them know when they're eligible for the upgrade.
```
Comment (mestew) [inline windows/whats-new/windows-11-plan.md:39]:
```suggestion
Microsoft is currently developing analysis tools to help you evaluate your devices against the Windows 11 hardware requirements. When Windows 11 reaches general availability, users running Windows 10 Home, Pro, and Pro for Workstations will be able to use the [PC Health Check](https://www.microsoft.com/windows/windows-11#pchealthcheck) app to determine their eligibility for Windows 11. Users running Windows 10 Enterprise and Education editions should rely on their IT administrators to let them know when they're eligible for the upgrade.  
```

File: windows/whats-new/windows-11-plan.md [hunk 5]
```diff
@@ -38,15 +38,15 @@ As a first step, you'll need to know which of your current devices meet the Wind
...
  
-Enterprise organizations looking to evaluate device readiness in their environments can expect this capability to be integrated into existing Microsoft tools, such as Endpoint analytics and Update Compliance. This capability will be available when Windows 11 is generally available. Microsoft is also working with software publishing partners to facilitate adding Windows 11 device support into their solutions. 
+**Enterprise organizations** looking to evaluate device readiness in their environments can expect this capability to be integrated into existing Microsoft tools, such as Endpoint analytics and Update Compliance. This capability will be available when Windows 11 is generally available. Microsoft is also working with software publishing partners to facilitate adding Windows 11 device support into their solutions.
```
Comment (mestew) [inline windows/whats-new/windows-11-plan.md:41]:
```suggestion
Enterprise organizations looking to evaluate device readiness in their environments can expect this capability to be integrated into existing Microsoft tools, such as Endpoint analytics and Update Compliance. This capability will be available when Windows 11 is generally available. Microsoft is also working with software publishing partners to facilitate adding Windows 11 device support into their solutions. 
```

File: windows/whats-new/windows-11-plan.md [hunk 6]
```diff
@@ -42,11 +42,11 @@ Enterprise organizations looking to evaluate device readiness in their environme
 
 ## Windows 11 availability
 
-The availability of Windows 11 will vary according to a device's hardware and whether the device receives updates directly, or from a management solution that is maintained by an IT administrator. 
+The availability of **Windows 11** will vary according to a device's hardware and whether the device receives updates directly, or from a management solution that is maintained by an IT administrator.
```
Comment (mestew) [inline windows/whats-new/windows-11-plan.md:45]:
```suggestion
The availability of Windows 11 will vary according to a device's hardware and whether the device receives updates directly, or from a management solution that is maintained by an IT administrator. 
```

**Comment by mestew** [comment]
Did some style and URL edits.

**Pull request merged**